inventory/group_vars/all.yml

```diff
@@ -51,11 +51,14 @@ cluster_name: cluster.local
 # but don't know about that address themselves.
 # access_ip: 1.1.1.1
 
-# Choose network plugin (calico, weave or flannel)
-kube_network_plugin: flannel
+# Choose network plugin (calico, weave, flannel or fan)
+kube_network_plugin: fan
 
 # Kubernetes internal network for services, unused block of space.
-kube_service_addresses: 10.233.0.0/18
+kube_service_addresses: 10.12.1.0/24
+
+# example underlay for ubuntu fan network
+#kube_network_fan_underlay: 10.11.0.0/16
```

Comment:
You need to set this - wasn't quite sure how this is supposed to be "detected", since it's a `/16` you pick from your own network. I noticed there was a bunch of other plugin specific variables, so opted for this one.